src/games/strategy/engine/history/SerializationWriter.java

```diff
@@ -0,0 +1,7 @@
+package games.strategy.engine.history;
+
+import java.io.Serializable;
+
+public interface SerializationWriter extends Serializable {
```

Comment:
Can we reduce teh scope of this interface? Was this actually causing warnings? A small 'inner' interface is not necessarily a bad thing. For classes with logic, their own class file I think is almost always good, but here, not necessarily convinced.

Failing the above, adding in a class javadoc to describe how this interface is used, or should be used, woudl be of help.